lib/matplotlib/backends/qt_editor/formlayout.py

```diff
@@ -123,7 +120,8 @@ def __init__(self, color, parent=None):
     def update_color(self):
```

Comment:
```
    def update_color(self):
        # to_qcolor defaults to black for invalid inputs.
        self.colorbtn.color = to_qcolor(self.text())
```
(actually it's not clear at all where the "defaults to black" happens, I just took a random guess)